api-tests/features/p2-web-journey.feature

```diff
@@ -188,3 +188,71 @@ Feature: P2 Web document journey
     Then I get an OAuth response
     When I use the OAuth response to get my identity
     Then I get a 'P2' identity
+
+  Rule: User drops out of KBV CRI via thin file or failed checks
+    Background: Navigate to KBV CRI
+      When I submit a 'ukPassport' event
+      Then I get a 'ukPassport' CRI response
+      When I submit 'kenneth-passport-valid' details to the CRI stub
+      Then I get an 'address' CRI response
+      When I submit 'kenneth-current' details to the CRI stub
+      Then I get a 'fraud' CRI response
+      When I submit 'kenneth-score-2' details to the CRI stub
+      Then I get a 'page-pre-experian-kbv-transition' page response
+      When I submit a 'next' event
+      Then I get a 'kbv' CRI response
+
+    Scenario Outline: <scenario> - user is able to receive identity via DCMAW
+      When I submit '<kbv-details>' details to the CRI stub
+      Then I get a 'pyi-cri-escape' page response
+      When I submit an 'appTriage' event
+      Then I get a 'dcmaw' CRI response
+      When I submit 'kenneth-passport-valid' details to the CRI stub
+      When I submit a 'next' event
+      Then I get an OAuth response
+      When I use the OAuth response to get my identity
+      Then I get a 'P2' identity
+
+      Examples:
+      | scenario      | kbv-details       |
+      | KBV thin file | kenneth-thin-file |
+      | Failed KBV    | kenneth-score-0   |
```

Comment:
As discussed with Patrick - dropout and thin file are actually both just 'score 0', so there's no need to distinguish these :)

We can distinguish them for HMRC/DWP KBVs (but that's not in scope of this ticket!)